server/src/controllers/userController.ts

```diff
@@ -1,8 +1,7 @@
 import { client } from "../redis";
 
-async function getUser(utorid: any) {
-  if (utorid === null || utorid === undefined || typeof utorid !== "string")
-    return { status: 400, data: { message: "Invalid utorid" } };
+async function getUser(utorid: string) {
+  if (!utorid) return { status: 400, data: { message: "Invalid utorid" } };
```

Comment:
If `utorid` is never undefined then we don't really need this if statement right?